workspaces/core/src/search/private/components/Input.tsx

```diff
@@ -0,0 +1,90 @@
+import React, { forwardRef, Fragment, Ref } from "react";
+import { GenericSearch } from '@heathmont/moon-icons-tw';
+
+interface SearchProps {
+  onChange: (value: string) => void;
+  onFocus?: () => void;
+  onBlur?: () => void;
+  placeholder?: string;
+  prefix?: string[];
+  value: string;
+}
+
+const Close = () => <span className="text-trunks text-sm">Clear</span>
```

Comment:
I think we should do this `Clear` text as a prop from the outside, for example to give possibilities to translate it